x-pack/plugins/security_solution/common/endpoint/types/trusted_apps.ts

```diff
@@ -69,10 +69,14 @@ export enum ConditionEntryField {
   SIGNER = 'process.Ext.code_signature',
 }
 
+export enum OperatorEntryField {
+  included = 'included',
+  wildcard_caseless = 'wildcard_caseless',
+}
 export interface ConditionEntry<T extends ConditionEntryField = ConditionEntryField> {
   field: T;
   type: 'match';
-  operator: 'included';
+  operator: keyof typeof OperatorEntryField;
```

Comment:
You should not have the need to use `keyof typeof` here. Enum's can be used as straight types, so:

```suggestion
  operator: OperatorEntryField;
```

should suffice

Reply:
Oddly TS doesn't deconstruct the enum's keys.  Instead of inferring `operator: 'included | 'wildcard_caseless'` it infers `operator: OperatorEntryField`.